.infrastructure/helm-chart/todoapp/templates/app/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Chart.Name }}-pvc-data
+  namespace: {{ .Values.common.namespace }}
```

Comment:
Ensure that `.Values.common.namespace` is defined in your `values.yaml` file. If it's not set, the PVC might not be created in the intended namespace.